library/monasca_notification_method.py

```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/python
+# ©Copyright 2015 Hewlett-Packard Development Company, L.P.
```

Comment:
This failed on my mini-mon:
failed: [mini-mon] => {"failed": true, "parsed": false}
  File "/home/vagrant/.ansible/tmp/ansible-tmp-1439572733.01-189664143461273/monasca_notification_method", line 2
SyntaxError: Non-ASCII character '\xc2' in file /home/vagrant/.ansible/tmp/ansible-tmp-1439572733.01-189664143461273/monasca_notification_method on line 2, but no encoding declared; see http://www.python.org/peps/pep-0263.html for details

FATAL: all hosts have already failed -- aborting

I think it did not like the copyright symbol. I'm guessing that will also be a problem in monasca_alarm_definition.py